apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -282,6 +294,20 @@ static void tc_task_task_restart(void)
 	int ret_chk;
 	unsigned int remain;
 	g_callback = 0;
+
+	/* Check for NULL pid parameter  */
+
+	pid = task_create("tc_task_re", SCHED_PRIORITY_MAX - 1, 1024, restart_task, (char * const *)NULL);
+	TC_ASSERT_GT("task_create", pid, 0);
+
+	g_callback = 0;
+	ret_chk = task_restart(0);
```

Comment:
To test restart functionality, below looks like correct.
1. launch A task
2. finish A task
3. restart A or test incorrect arg

But, there is no 2nd step in yours.
Even though ```restart_task``` is alive, you execute ```task_restart``` function. 
I think it is not correct.

Reply:
done, removed sleep(SEC_1) from restart_task entry point function